renderer/pyscript-offline/public/README.md

```diff
@@ -0,0 +1,19 @@
+# UNLWS Renderer
+A tool to specify UNLWS texts in code and display them.
+
+## Setup
+
+1. Clone the repo
+2. Open a terminal in `...\unlws\renderer\pyscript-offline\public`
+3. Start a local web server. If you have Python installed, this can be done by running
+    ```sh
+    python3 -m http.server
+    ```
+    or
+    ```sh
+    python -m http.server
+    ```
+4. Go to `http://localhost:8000` in your browser
+
+Each time the site is loaded, it will download a WebAssembly-based Python runtime (via PyScript), so expect to wait a while.
```

Comment:
Not a negative for your readme, but how can prevent it going to external sites for this? I'm not entirely clear on how to cache pyscript libraries into the repo.